### PR TITLE
[backend/frontend] Merge tabs in scenario form (#1822)

### PR DESCRIPTION
**Pull request by johanah29**

### CODE REVIEW

**Comment by johanah29** [comment]
#1822

File: openbas-front/src/admin/components/scenarios/ScenarioCreation.tsx [hunk 1]
```diff
@@ -38,6 +43,29 @@ const ScenarioCreation: FunctionComponent<Props> = ({
     );
   };
 
+  const { settings }: { settings: PlatformSettings } = useHelper((helper: LoggedHelper) => ({
+    settings: helper.getPlatformSettings(),
+  }));
+  useDataLoader(() => {
+    dispatch(fetchPlatformParameters());
```
Comment (guillaumejparis) [inline openbas-front/src/admin/components/scenarios/ScenarioCreation.tsx:50]: Question : Do we really need to refetch PlatformParameters ?

File: openbas-front/src/admin/components/scenarios/ScenarioCreation.tsx [hunk 2]
```diff
@@ -38,6 +43,29 @@ const ScenarioCreation: FunctionComponent<Props> = ({
...
+    scenario_category: 'attack-scenario',
+    scenario_main_focus: 'incident-response',
+    scenario_severity: 'high',
+    scenario_subtitle: '',
+    scenario_description: '',
+    scenario_external_reference: '',
+    scenario_external_url: '',
+    scenario_tags: [],
+    scenario_message_header: 'SIMULATION HEADER',
+    scenario_message_footer: 'SIMULATION FOOTER',
```
Comment (guillaumejparis) [inline openbas-front/src/admin/components/scenarios/ScenarioCreation.tsx:64]: todo: translate these

File: openbas-api/src/main/java/io/openbas/rest/scenario/form/ScenarioInput.java [hunk 3]
```diff
@@ -45,4 +47,19 @@ public class ScenarioInput {
 
   @JsonProperty("scenario_tags")
   private List<String> tagIds = new ArrayList<>();
+
+  @JsonProperty("scenario_mail_from")
+  @Email
+  @NotBlank
+  private String from;
+
+  @Column(name = "scenario_reply_to", nullable = false)
```
Comment (savacano28) [inline openbas-api/src/main/java/io/openbas/rest/scenario/form/ScenarioInput.java:56]: This @Column annotation is not used in a DTO

File: openbas-model/src/main/java/io/openbas/database/model/Scenario.java [hunk 4]
```diff
@@ -111,11 +111,11 @@ public enum SEVERITY {
 
   @Column(name = "scenario_message_header")
   @JsonProperty("scenario_message_header")
-  private String header = "EXERCISE - EXERCISE - EXERCISE";
+  private String header = "EXERCISE HEADER";
```
Comment (guillaumejparis) [inline openbas-model/src/main/java/io/openbas/database/model/Scenario.java:114]:
```suggestion
  private String header = "SIMULATION HEADER";
```

File: openbas-model/src/main/java/io/openbas/database/model/Scenario.java [hunk 5]
```diff
@@ -111,11 +111,11 @@ public enum SEVERITY {
...
 
   @Column(name = "scenario_message_footer")
   @JsonProperty("scenario_message_footer")
-  private String footer = "EXERCISE - EXERCISE - EXERCISE";
+  private String footer = "EXERCISE FOOTER";
```
Comment (guillaumejparis) [inline openbas-model/src/main/java/io/openbas/database/model/Scenario.java:118]:
```suggestion
  private String footer = "SIMULATION FOOTER";
```

**Review by guillaumejparis — APPROVED** [review]

**Pull request merged**